.eslintrc.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
@KaveriMoger do you think we should use yaml over json.
which one do you think is better?